inc/Form/DropdownElement.php

```diff
@@ -91,18 +123,57 @@ public function attr($name, $value = null) {
     public function val($value = null) {
         if($value === null) return $this->value;
 
-        if(isset($this->options[$value])) {
+        $value_exists = $this->setValueInOptGroups($value);
+
+        if($value_exists) {
             $this->value = $value;
         } else {
             // unknown value set, select first option instead
-            $keys = array_keys($this->options);
-            $this->value = (string) array_shift($keys);
+            $this->value = $this->getFirstOption();
+            $this->setValueInOptGroups($this->value);
```

Comment:
We are not, we are throwing an exception when setting the `multiple`-attribute. I removed my erroneous handling of multi selects in  45082b9.
We should consider a new PR for adding multi-select-functionality.